setup.py

```diff
@@ -15,6 +15,11 @@
 for entry in dependency_links:
     os.system("pip install {}".format(entry))
 
-setup(name='humblerl',
+setup(name='HumbleRL',
       version='0.0.1',
+      description='Straightforward reinforcement learning Python framework',
+      author='Piotr Januszewski',
+      author_email='piojanuszewski@gmail.com',
```

Comment:
Plz nie xD Nie udostępniajmy mojego maila 😄 Już ostatnio zacząłem dostawać spam-u sporo 😮 Odpuśćmy to sobie, albo musimy założyć jakiegoś maila na takie rzeczy.